exporter/azuremonitorexporter/trace_to_envelope.go

```diff
@@ -300,7 +300,7 @@ func fillRequestDataHTTP(span ptrace.Span, data *contracts.RequestData) {
 	sb.Reset()
 
 	/*
-		To construct the value for data.Url we will use the following sets of attributes as defined by the otel spec
```

Comment:
This change and the other one in azuremonitorexporter are not related to the PR AFAICT, could we remove them?

Reply:
oh intellij was too eager in renaming, sorry, I'll revert.